jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -107,11 +73,12 @@ def test_no_spec_no_flux_uncert(specviz_app, spectrum1d):
 
 
 def test_no_spec_no_flux_uncert_unit_exp_none(specviz_app, spectrum1d):
+    np.random.seed(42)
```

Comment:
Need to set a seed in all functions that you call `np.random`. I'd rather you not use it, but since you made your choice, I'll respect your decision.